tests/e2e/test_ironman.py

```diff
@@ -132,19 +138,134 @@ def test_exec_succ_pod(tmpdir):
         o.delete_pod()
 
 
+def test_local_path_e2e_inside_w_exec(tmpdir):
+    p = Path(tmpdir)
+    m_dir = MappedDir(str(p), "/sandcastle", with_interim_pvc=True)
+
+    p.joinpath("stark").mkdir()
+    p.joinpath("qwe").write_text("Hello, Tony!")
+
+    o = Sandcastle(
+        image_reference=SANDBOX_IMAGE,
+        k8s_namespace_name=NAMESPACE,
+        mapped_dirs=[m_dir],
+        working_dir="/sandcastle",
+    )
+    o.run()
+    try:
+        out = o.exec(command=["ls", "/sandcastle/qwe"])
+        assert "qwe" in out
+        o.exec(command=["touch", "/sandcastle/stark/asd"])
+        assert p.joinpath("stark/asd").is_file()
+        o.exec(command=["touch", "./zxc"])
+        assert p.joinpath("zxc").is_file()
+    finally:
+        o.delete_pod()
+
+
+def test_file_got_changed(tmpdir):
+    m_dir = MappedDir(str(tmpdir.mkdir("stark")), "/sandcastle", with_interim_pvc=True)
+
+    p = Path(m_dir.local_dir).joinpath("qwe")
+    p.write_text("Hello, Tony!")
+
+    o = Sandcastle(
+        image_reference=SANDBOX_IMAGE, k8s_namespace_name=NAMESPACE, mapped_dirs=[m_dir]
+    )
+    o.run()
+    try:
+        o.exec(command=["bash", "-c", "echo '\nHello, Tony Stark!' >>/sandcastle/qwe"])
+        assert "Hello, Tony!\nHello, Tony Stark!\n" == p.read_text()
+    finally:
+        o.delete_pod()
+
+
+def test_md_e2e(tmpdir):
+    t = Path(tmpdir)
+    m_dir = MappedDir(str(t), "/sandcastle", with_interim_pvc=True)
+
+    run_command(
+        ["git", "clone", "https://github.com/packit-service/hello-world.git", t]
```

Comment:
Will this work with the `Path` object? `t` -> `str(t)`?

Reply:
tests are passing so it does work apparently